fastify/website-next
# v4 documentation page

closes #27 


## how to test it

1. install GH CLI https://cli.github.com/
2. install the repo as described in the readme
3. run `npm run build:website`
4. run `npm start` to see the documentation

## features

- [x] it supports legacy `/docs/latest` links
- [x] every v4's minor version has its own documentation
- [x] the left menu is equal to the previous website
- [x] all the pages are visible correctly
- [x] all links are working
- [x] the edit page button works
- [x] it supports `/docs/master` links. I will add a redirect to `/docs/latest` or `/docs/next`
